app/views/dashboard/_links.njk

```diff
@@ -1,4 +1,6 @@
-{% if linksToDisplay.length % 3 == 0 %}
+
+{% set noOfLinks = linksToDisplay.length + (1 if requestedStripeTestAccount else 0) %}
```

Comment:
Would this be slightly simpler if we sent the `requestPspTestAccount` in the linksToDisplay when test account is requested, but then checked `requestedStripeTestAccount` value to check what content to display later? This would be closer to how it works for the go live links.

We are displaying the "link", it just has different content and isn't a link.

Reply:
Thanks, updated!